common/components/SubTabs/index.tsx

```diff
@@ -1,62 +1,38 @@
 import React from 'react';
-import classnames from 'classnames';
-import { TranslateType } from 'translations';
-import { Link } from 'react-router-dom';
+import { NavLink, RouteComponentProps } from 'react-router-dom';
 import './SubTabs.scss';
 
-export interface Tab<SubTabProps = null> {
+interface Tabs {
+  name: string | React.ReactElement<any>;
   path: string;
-  name: TranslateType;
-  isDisabled?(props: SubTabProps | {}): boolean;
```

Comment:
Just a heads up, we do still need this prop.

Reply:
Ah, I see you've added `disabled`. You'll have to dig into each usage of the subtab and pass the computed value instead of the function.

Not sure this change is useful / necessary in the scope of this PR.

Reply:
The only subtab using `disabled` is send. I've updated the pr to handle disabled subroutes & tabs

Reply:
Whichever get's merged first can handle the conflict, it's not a big change.